src/presenters/form.js

```diff
@@ -22,7 +22,10 @@ const props = [
   'version',
   'keyId',
   'sha256',
+  // Form and form draft, not form version
   'enketoId',
+  // Form only, not form draft or form version
+  'enketoSingleId',
```

Comment:
Frontend uses the same property as the Backend response, which is `enketoSingleId`, so I think changing this would require a Backend change. But I like the sound of `enketoOnceId`: it feels like a nice parallel with the `once` property of public links.